www/base/src/components/MultipleWorkersActionsModal/MultipleWorkersActionsModal.tsx

```diff
@@ -36,6 +37,42 @@ const workerToSelectOption = (worker: Worker): SelectOption => {
   return { value: worker, label: worker.name }
 }
 
+const Control = (props: any) => {
```

Comment:
Props should be typed. Probably makes sense to check what props `components.Control` accepts and then add additional members to it.